.travis.yml

```diff
@@ -31,7 +31,8 @@ env:
   - INCLUDE="test_module,second_module" UNIT_TEST="1"
   - VERSION="7.0" INCLUDE="test_module,second_module" ODOO_REPO="OCA/OCB"  # ODOO_REPO usage example
   - VERSION="6.1" INCLUDE="test_module,second_module"
-  - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="17"
+  - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="17" TRAVIS_PULL_REQUEST="false" # Use main pylint config file
+  - LINT_CHECK="1" TESTS="0" PYLINT_EXPECTED_ERRORS="18" TRAVIS_PULL_REQUEST="true"  # Use PR pylint config file
```

Comment:
Sorry, I'm talking about of TravisCI variable: `TRAVIS_PULL_REQUEST`
![travis_pr](https://cloud.githubusercontent.com/assets/6644187/9591467/21aafa74-5001-11e5-9959-42794e05375d.png)

I'm confunsing, can you help me to identify:
What `not-TravisCi` variable are you talking?